.github/workflows/gh-ci-tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: mda_gh_ci_tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.OS }}-latest
```

Comment:
Same question - are we expecting a matrix here?

Reply:
Fixed.